sgt-back-end/server.js

```diff
@@ -0,0 +1,152 @@
+import pg from 'pg';
+import express from 'express';
+
+const app = express();
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+app.get('/api/grades', async (req, res) => {
+  try {
+    const sql = `
+      select *
+        from "grades"
+    `;
+    const result = await db.query(sql);
+    const allGrades = result.rows;
```

Comment:
Or you could get rid of the `allGrades` local variable and just pass `result.rows` directly into `.json`.